README.md

```diff
@@ -1,39 +1,66 @@
-# Temporal  
 [![Build status](https://badge.buildkite.com/fc0e676d7bee1a159916af52ebdb541708d4b9f88b8a980f6b.svg)](https://buildkite.com/temporal/temporal-server)
 [![Coverage Status](https://coveralls.io/repos/github/temporalio/temporal/badge.svg?branch=master)](https://coveralls.io/github/temporalio/temporal?branch=master)
 [![Discourse](https://img.shields.io/static/v1?label=Discourse&message=Get%20Help&color=informational)](https://community.temporal.io)
 
-Visit [docs.temporal.io](https://docs.temporal.io) to learn about Temporal.
+# Temporal  
+
+Temporal is a microservice orchestration platform which enables developers to build scalable applications without sacrificing productivity or reliability.
+Temporal server executes units of application logic, Workflows, in a resilient manner that automatically handles intermittent failures, and retries failed operations.
 
-This repo contains the source code of the Temporal server. To implement workflows, activities and worker use [Go SDK](https://github.com/temporalio/sdk-go) or [Java SDK](https://github.com/temporalio/sdk-java).
+The programming model of Temporal exposed via the SDKs (currently for Java and Go) offers developers the simple abstractions of Workflows and Activities that can virtually "resume" execution after a failure from the same line of code.
 
-See Maxim's talk at [Data@Scale Conference](https://atscaleconference.com/videos/cadence-microservice-architecture-beyond-requestreply) for an architectural overview of Temporal.
+Temporal is a mature technology, a fork of Uber's Cadence.
+Temporal is being developed by [Temporal Technologies](https://temporal.io/), a startup by the creators of Cadence.
+
+[![Temporal](video.png)](http://www.youtube.com/watch?v=f-18XztyN6c "Temporal")
+
+Learn more about Temporal at [docs.temporal.io](https://docs.temporal.io).
 
 ## Getting Started
 
-### Start the temporal-server locally
+### Download and Start Temporal Server Locally
+
+Execute the following commands to start a pre-built image along with all the dependencies.
 
-We highly recommend that you use [Temporal service docker](docker/README.md) to run the service.
+```bash
+$ curl -L https://github.com/temporalio/temporal/releases/latest/download/docker.tar.gz | tar -xz
+$ cd docker
+$ docker-compose up
+```
+
+Refer to [Temporal service docker](docker/README.md) for more advanced options.
 
 ### Run the Samples
 
-Try out the sample recipes for [Go](https://github.com/temporalio/samples-go) or [Java](https://github.com/temporalio/samples-java) to get started.
+Clone or download samples for [Go](https://github.com/temporalio/samples-go) or [Java](https://github.com/temporalio/samples-java) and try to run them against the local Temporal server.
+We have a number of [HelloWorld type scenarios](https://github.com/temporalio/samples-java#helloworld) available.
```

Comment:
There is "hello world" in go too.

Reply:
But only one. Is it worth linking to it from here?

Reply:
I rephrased a bit.